Translate slides from 1 to 5 (#4)
xsuchy/ada-zangemann-cz: Ada_Zangemann-cs.txt

[[I:ada-p03
IMAGE: Matthias Kirschner, Sandra Brandstätter - ADA & ZANGEMANN -
Příběh o softwaru, skateboardech a malinové zmrzlině.
Titulek uprostřed, autoři nahoře vlevo.
Ve spodu vpravo je Adin pes s šedivou srstí, několika bílými skvrnami
a červeným obojkem se svítícími žlutými LED, který vrtí ocasem.
]]
Matthias Kirschner
Sandra Brandstätterová

ADA & ZANGEMANN

Příběh o softwaru, skateboardech a malinové zmrzlině.

## 4 ##
==== Slide 2

> [B]yla jednou jedna malá holčička jménem Ada. Její rodina byla tak chudá, že všechny jejich úspory se vešly do plechovky od sušenek. Neměli peníze ani na to, aby mohli bydlet v opravdovém bytě. Místo toho bydlely Ada, její maminka a její malý bratr Alan v chatrči poblíž vrakoviště na kraji města.

Reviewer: ```suggestion
[B]yla jednou jedna malá holčička jménem Ada. Její rodina byla tak chudá, že všechny jejich úspory se vešly do plechovky od sušenek. Neměli peníze ani na to, aby mohli bydlet v opravdovém bytě. Místo toho bydlela Ada se svou maminkou a malým bratrem Alanem v chatrči poblíž vrakoviště na kraji města.
```
Přeformulovala bych tu poslední větu, protože mi není zřejmé, čí bratr Alan je, jestli Ady nebo maminky, když se tam opakuje to "její". Plus by mělo být "bydleli", protože v několikanásobném podmětu je i Alan, ale v navrhované verzi už je to jedno.

Author: Klidně to tak může být, jenom bych rád podotknul, že -y  je zde správně, viz https://prirucka.ujc.cas.cz/?id=601, část 2.1, odstavec 2.

Reviewer: Jo, máš pravdu, zapomněla jsem, že je možné obojí, pokud je přísudek před podmětem.

V pozadí je Adina chatrč a maminka s bratrem, kteří ji mávají na rozloučenou.
Na zemi před nimi sedí pes.
Alan sedí na dvou schodcích do domku. Má krátké tmavé vlasy a je oblečený
do bílého trička s krátkými rukávy olemovanými červeným lemem a dvěma červenými
pásky na spodním okraji trička. Alan má modré kraťasy a je naboso.
Adina máma má tmavou rtěnku a černé vlasy svázané bílou stuhou. Má krátké
modré tričko s bílým zakončením, bílé kalhoty a žluté boty.
Adina chatrč je dřevěná s bílou přední stěnou. Střecha je z tmavě červených
vlnitých plechů. Vpředu jsou kromě dveří ještě dvě okna a další okno je na
levé straně. Na střeše stojí žlutá kachna. Střechou se prodírá bílý komín.
Na anténě na levé straně střechy je uvázaná šňůra na prádlo, na které visí
zelený top, světle modré kalhoty a bílé kalhotky, přichycené červenými kolíčky na prádlo.
Napravo od domu je veranda se solárním panelem. Pod verandou je matrace, červený sud od
oleje a šedý kufr.
Za domem jsou hromady starých věcí, na kterých můžeme rozeznat pračku, auto,
a mnoho dřevěných prken.
]]

** 5 **

[[I:ada-p05
IMAGE: Prestižní časopisy (“Hvězdy“, “Peníze”, “Největší boháč”) na kterých
je Zangemann a jeho vila.
]]

> Daleko odtud, na opačném konci, žil slavný vynálezce Zangemann. Byl nesmírně bohatý. Žádný list papíru na světě nebyl dost velký na to, aby se na něj vešel zůstatek jeho účtu se všemi těmi číslicemi a nulami. Bydlel v obrovském domě s bazénem a tobogánem, mnoha schodišti, věžemi, stovkami oken a tolika pokoji, že i on sám se v nich pravidelně ztrácel. Zangemannovo sídlo se nacházelo vysoko na kopci. Celé město měl pod sebou jako na dlani.

Reviewer: ```suggestion
Daleko odtud, na opačném konci města, žil slavný vynálezce Zangemann. Byl nesmírně bohatý. Žádný list papíru na světě nebyl dost velký na to, aby se na něj vešel zůstatek jeho účtu se všemi těmi číslicemi a nulami. Bydlel v obrovském domě s bazénem a tobogánem, mnoha schodišti, věžemi, stovkami oken a tolika pokoji, že i on sám se v nich pravidelně ztrácel. Zangemannovo sídlo se nacházelo vysoko na kopci. Celé město měl pod sebou jako na dlani.
```

Author: Nad tím jsem přemýšlel hodně. Představuju si, jak by to mělo vypadat, kdyby text plynul v normální knize bez obrázků. Tam by bylo naprosto jasné, že se mluví o konci města, protože text jde takto:

_... na vrakovišti na konci města. Daleko odtud, na opačném konci, žil slavný ..._

Originální text velmi trpí na opakovaná slova a stylisticky to určitě není Shakespear. Snažme se o co nejlepší čtivost v češtině.

Reviewer: Jako jo, pokud by to bylo ve stejné kapitole, tak jo, ale předpokládala jsem, že nový slide je jako nová kapitola, takže najednou chybí kontext. Tady to teda hodně záleží na tom, jak moc jsou oddělené ty slidy.

Author: Tak mám za to, že to bude vytištěná kniha, takže bych čekal, že čtenář půjde po tom textu. Počkejme, co řekne @xsuchy.

Author: Upřímná otázka. Myslíš, že kdyby to i bylo na dvou slidech, tak bys nebyla ten kontext schopná udržet? Protože já věřím, že ano.

Reviewer: Nevím, přijde mi, že je to hodně o tom, jak vypadá ta výsledná knížka, a nemám moc zkušeností se čtením pro děti. Představuji si, že dítěti přečteš kousek, pak se spolu chvilku díváte na obrázek a pak na další stránce pokračujete. A v ten moment ten kontext chybí. Ale nechala bych toto rozhodnutí na někom jiném.

Reviewer 2: Ano, počítal bych s tím, že výsledek bude fyzická kniha. Pokud ji chcete vidět fyzicky stavte se v kanclu kde jsem ji v knihovně nechal. 
Jak to bude vypadat si muzete vyzkouset podle tohoto navodu https://github.com/xsuchy/ada-zangemann-cz/tree/main/scripts#usage
Cilem je IMHO cilovka deti kolem 5-8 roku. A nebál bych se vzít si za vzor Čtvrtka a jeho Rumcajsa, který ty věty sekal extrémně krátké. Takže... co vlastně... asi jsem pro to nebát se opakovat slova. Stejně pak budeme dělat zkušební čtení, tak to kdyžtak upravíme později.

## 6 ##
==== Slide 3
Už v dětském věku Zangemanna fascinovaly počítače. Tehdy to ještě byly obrovské stroje s mnoha kabely a hlučnými ventilátory. Ve škole malý Zangemann často snil o tom, co by všechno s počítači dělal, kdyby byly jen o něco menší – dost malé na to, aby je mohl zabudovat do nejrůznějších zařízení. Jako první by zabudoval počítač do svého skateboardu, který by tak při jízdě vydával super zvuky, třeba jako siréna hasičů nebo start rakety. A vymyslel by stroj na zmrzlinu! Počítač by dokázal namíchat

> ty nejlepší příchutě a zmrzlinu by zároveň i prodával. Jeho stroje by stály na každém rohu a on by si tak skočit pro svou oblíbenou zmrzlinu, kdykoli by měl chuť. Kromě toho by postavil robota na úklid a stroj na třídění stavebnic, aby měl ve svém pokoji vždy krásně uklizeno. Zangemann měl každý den nové nápady. Nemohl myslet na nic jiného.

Reviewer: ```suggestion
ty nejlepší příchutě a zmrzlinu by zároveň i prodával. Jeho stroje by stály na každém rohu a on by si tak mohl skočit pro svou oblíbenou zmrzlinu, kdykoli by měl chuť. Kromě toho by postavil robota na úklid a stroj na třídění stavebnic, aby měl ve svém pokoji vždy krásně uklizeno. Zangemann měl každý den nové nápady. Nemohl myslet na nic jiného.
```

[[I:ada-p08b
OBRÁZEK: Zangemannova originální zmrzlinovačka.
Je světle modrá, podobná automatu na nápoje, ale místo kelímku je zde kornoutek, který se plní zmrzlinou. Nad dávkovačem je monitor, který zobrazuje druh zmrzliny, který se právě podává. V tomto případě je to ananasová zmrzlina.
Vlevo se nachází otvor pro vkládání mincí, různé tlačítka pro výběr příchutě a otvor pro vracení mincí.
Nahoře je kulatý bílý znak s kopečkem cukrově růžové zmrzliny v kornoutku.
Na zadní straně stroje je chladicí zařízení s písmenem „Z“ jako logem Zangemanna.
]]
Dospělí i děti jeho vynálezy milovali. Děti chtěly mít Zangemannův skateboard a a k němu pokaždé ten nejnovější zvuk, aby se s ním mohly chlubit kamarádům na školním dvoře. Mnozí měli také jeho praktické reproduktory. Stačilo do nich jen uvést název písně a ta se hned začala přehrávat. Komu ještě nějaké kapesné zbylo, mohl si odpoledne koupit automaticky namíchanou zmrzlinu z originální Zangemannovy zmrzlinovačky. Vypadalo to jako kouzla, ale vše zařizovaly malé počítače, které Zangemann do svých přístrojů vestavěl.
[[I:ada-p08a

> OBRÁZEK: Originální Zangemannův elektrický skateboard zanechávající modrou stopu s notami a barevnými kruhy. Z dítěte, které jede na skateboardu, jsou vidět jenom koleny, bílé ponožky a olivově zelené boty s bílou podrážkou.

Reviewer: ```suggestion
OBRÁZEK: Originální Zangemannův elektrický skateboard zanechávající modrou stopu s notami a barevnými kruhy. Z dítěte, které jede na skateboardu, jsou vidět jenom kolena, bílé ponožky a olivově zelené boty s bílou podrážkou.
```

]]

** 9 **

[[I:ada-p09
OBRÁZEK: Ada kutí s předměty ve své improvizované laboratoři.

> Ada sedí na převrácené bedně s vodou a pracuje na opravě s pomocí žlutého šroubováku. Na stole jsou také zelené kleště, červené kladivo, bílá prodlužovací šňůra a další přístroj. Ada pracuje u stolu, který se skládá z dřevěné desky položené na dvou olejových sudech (jednom šedém a jednom červeném) a malém dřevěném kozlíku. Za Adou je pračka, potrubí a deska. Před improvizovaným stolem jsou pneumatiky a starý telefon s vadným drátem. Alan přichází k Adě a nad hlavou drží žlutý větrák. Alan má nyní na sobě bílé ponožky a žluté boty.

Reviewer: Nevidím ten obrázek, tak nevím, ale fakt jsem se zarazila u "sedí na převrácené bedně s vodou" a přemýšlela, jak to, že se ta voda nevylije, když je bedna převrácená. Při přečtení anglické verze si představím spíš převrácený kanystr na vodu (ne nutně s vodou). Nevím, jestli jsou někde obrázky zveřejněny, ale podle popisu jsem našla https://pedagogie.ac-clermont.fr/numerique-educatif/wp-content/uploads/sites/6/2024/10/ADA-2.jpg a podle něj bych to popsala spíš jako ten kanystr na vodu.

Author: Ok, to dává smysl.

Reviewer: Souhlasím, přepravka je lepší. Asi i na tom obrázku je přepravka, jen mi to nedocvaklo.

Reviewer 2: Obrazky jsou zde a konkretne tento https://github.com/xsuchy/ada-zangemann-cz/blob/main/illustrations/common/ada-p04.png
